android/src/main/java/io/ably/flutter/plugin/AblyEventMessage.java

```diff
@@ -0,0 +1,15 @@
+package io.ably.flutter.plugin;
+
+class AblyEventMessage<T> {
+    final String eventName;
+    final T message;
+
+    AblyEventMessage(final String eventName, final T message) {
+        if (null == eventName) {
+            throw new NullPointerException("message cannot be null.");
+        }
+        this.eventName = eventName;
+        this.message = message;
+    }
+
+}
```

Comment:
Missing EOF

Reply:
c585daabfbf0eec1269813774fb37bcf41f0b6f7 contains new extra line and log message fix